extensions/azurecore/src/azureResource/commands.ts

```diff
@@ -231,8 +231,8 @@ export function registerAzureResourceCommands(appContext: AppContext, tree: Azur
 			return;
 		}
 		// Ensure connection is saved to the Connections list, then open connection dialog
-		let connectionProfile = Object.assign({}, treeItem.payload, { saveProfile: true });
-		const conn = await appContext.apiWrapper.openConnectionDialog(undefined, connectionProfile, { saveConnection: true, showDashboard: true });
+		let connectionProfile = Object.assign({}, treeItem.payload, { saveProfile: false });
```

Comment:
Why do you consider this a bug? It seems like giving users a way to specifically add a connection to the Servers view from the Azure view is useful to have and this action should only happen if they specifically click the "Connect" action.

Reply:
This change was suggested by @kburtram

Reply:
The change I was suggesting was for the bug where connections are being adding to the servers view as a byproduct of using them in Azure view.  By using, I mean expanding and authentication, etc.  Those actions should not result in a connection being generated in the Server view.  

If the user explicitly clicked that "Add Connection" button next to the server\database node then in that scenario a connection should be added to servers.